tools/gatk_splitncigarreads.cwl

```diff
@@ -5,21 +5,21 @@ requirements:
   - class: ShellCommandRequirement
   - class: InlineJavascriptRequirement
   - class: ResourceRequirement
-    ramMin: 8000
-    coresMin: 4
+    ramMin: 16000
+    coresMin: 8
   - class: DockerRequirement
     dockerPull: 'kfdrc/gatk:4.1.7.0R'
 baseCommand: [/gatk, SplitNCigarReads]
 arguments:
   - position: 1
     shellQuote: false
     valueFrom: >-
-      --java-options "-Xmx7500m
+      --java-options "-Xmx16G
```

Comment:
should you set the limit such that it doesn't ever get to 16 and cause an error?

Reply:
Yeah, I just recently changed this in a commit. I thought there was a difference between `g` and `G` and thought I could get away with it. I was wrong. Ended up also increasing mem since it's seems a little hungry